rfcs/text/remove-classic.md

```diff
@@ -0,0 +1,54 @@
+- Start Date: 2020-03-12
+
+# Summary
+
+The Carbon core team intend to drop support for the Classic theme.
+
+# Motivation
+
+The Design Language System (DLS) has replaced most if not all of our "Classic" components.
+
+Currently we:
+- support the classic props with the modern themes.
+- support the classic props with the classic theme.
+
+This makes the simplest of changes time consuming and complex. This is evident in the most basic components like 
+`Button` where we have 13 props.
+
+# Detailed design
+
+1. We will remove all the classic examples from storybook as soon as possible.
+1. We will remove classic components/props piecemeal. Each of these will be a `BREAKING CHANGE`.
+1. When no components reference `classic` we can remove the theme.
+
+# Drawbacks
+
+- Many breaking changes will significantly increase the version number. This could look be interpreted as a unstable
```

Comment:
```suggestion
- Many breaking changes will significantly increase the version number. This wouldn't look great and could be interpreted as a unstable
```

Reply:
I don't want this RFC to say that increasing the version number won't look great as it is subjective. 

Lots of breaking changes mean that there is significant work happening, but also makes upgrading easier if they are small and frequent.

I'd like the reader to make their own mind on if this looks bad or not.

Reply:
Although I do see a typo.
```suggestion
- Many breaking changes will significantly increase the version number. This could be interpreted as a unstable
```